packages/toolkit/__tests__/build-project-global-css/__fixtures__/assets/css/frontend/styles.css

```diff
@@ -0,0 +1,10 @@
+html {
+	background: #f5f5f5;
+	padding: 20px;
+
+	@mixin margin-trim;
```

Comment:
@fabiankaegy I set up this test and for some reason, this line breaks building this file! It doesn't even create the css file.

Did I miss something? I'm wondering if this mixins is undefined and therefore breaking everyting. On the other hand I'd like to understand why this test doesn't even generate the css file with this line and it also doesn't give any error.

To run tests locally just do `npm run test -w=packages/toolkit -- --watch` you can run the tests then inspect the dist folder for each test to see what toolkit is outputting

Reply:
@nicholasio it appears that the issue was due to the `paths` not being set correctly for the `__fixtures__` path. See [52e5b67](https://github.com/10up/10up-toolkit/pull/389/commits/52e5b672a20209892a357b74a969cbbb1003bc89)

Reply:
That makes sense! Thanks!